server/router.go

```diff
@@ -77,8 +68,17 @@ func registerPlanEndpoints(plans *echo.Group, s *controllers.Server) {
 	// Gets the details of a plan by its UUID.
 	plans.GET("/:plan_id", s.GetPlanByID)
 
-	// Adds or updates the quota defaults of a plan.
-	plans.POST("/quota-defaults", s.AddPlanQuotaDefault)
```

Comment:
The old endpoint to add plan quota defaults was a little clunky, so I reimplemented it.